handbook/engineering/code-intelligence/goals.md

```diff
@@ -0,0 +1,45 @@
+# Cloud goals and priorities
+
+## Goals
+
+Track our progress in our [Looker dashboard](https://sourcegraph.looker.com/dashboards/131).
+
+### The majority of code intel results are precise (for languages we support fully)
```

Comment:
I don't feel strongly about this number so feel free to change it, but I think it is important to be more clear than fuzzy. You can either say, "all code intel results", or set some high percentage we want to attain. Majority, or most of is not clear enough to communicate the vision/intention.

```suggestion
### 95% of all code intel results are precise for any fully supported language
```

Reply:
We actually discussed this and our only concern was that we want to explicit state that we're aware of a very, very long tail that we don't really care to tackle. The long tail is composed of edge cases such as very particular deployment setups used by only one company.

95% seems fine as long as we see some wiggle room in the success criteria here.

Reply:
Ah, thanks that helps. In that case, I'd write it as explicitly as you stated. (Fill in the blank/tweak as needed):

```
### All code intel results are precise for any fully supported language

There is a long tail of edge cases for every language, and as described <where?>, we aspire to reach <some level> for every language. "All code intel results" is defined as all code intel results that meet this described level.
```

Reply:
We've consolidated the goals (and I don't think this applies anymore). Could you take another look?